src/pages/index.js

```diff
@@ -31,12 +31,12 @@ export default function Index() {
             <Promo title="React components" image={ComponentsImage} href="/components">
               Presentational UI components with encapsulated styles and constraint-based theme props.
             </Promo>
+            <Promo title="ViewComponents" image={ViewComponentsImage} href="/view-components">
+              ViewComponents for use in Rails applications.
```

Comment:
I can't make up my mind! How about:

```suggestion
              Build component-driven UI in Rails with ViewComponents.
```